fs/index.ts

```diff
@@ -63,3 +63,14 @@ export interface IClass {
   formatStartDate: string;
   _v?: number;
 }
+
+export interface IAnnouncements {
+  announcements: {
+    _id: string;
+    title: string;
+    description: string;
+    announcer: string;
+    date: Date;
+  }[];
+  isMore: boolean;
```

Comment:
what is isMore

Reply:
Whether or not there are more announcements to load in. On the site, there will be a button to load in more announcements if there are more to load. This way if there are too many announcements, the user isn't overloaded with info and their page isn't filled.